src/material-examples/autocomplete-optgroup/autocomplete-optgroup-example.ts

```diff
@@ -0,0 +1,115 @@
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormBuilder, Validators } from '@angular/forms';
+import { Observable } from 'rxjs/Observable';
+import { startWith } from 'rxjs/operators/startWith';
```

Comment:
In general we don't do spaces around the imports. E.g. these should be `import {startWith} from 'rxjs/operators/startWith';`